content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -0,0 +1,180 @@
+=======
+Belgium
+=======
+
+.. _belgium/introduction:
+
+Fiscal certification: POS restaurant
+====================================
+
+In Belgium, the owner of a cooking business such as a restaurant or food truck is required by law to
+use a Government-certified **Cash Register System** for their receipts. This applies if their yearly
+earnings (excluding VAT, drinks, and take-away food) exceed 25,000 euros.
+
+This Government-certified system entails the use of a **certified POS system**, along with a device
+called a **Fiscal Data Module** (or **black box**) and a **VAT Signing Card** (delivered after
+registration by the Service Public Federal Finances).
+
+A certified POS system
+----------------------
+
+The Odoo POS system is certified for the three latest major versions of databases hosted on **Odoo
+Online** and **odoo.sh**. To ensure that your POS system is certified, please refer to the
+following table.
+
++---------------+--------------------+--------------------+---------------------+
+|               |   **Odoo Online**  |     **odoo.sh**    | **Odoo On-Premise** |
++---------------+--------------------+--------------------+---------------------+
+| **Odoo 16.0** | Review in progress | Review in progress | Not certified       |
++---------------+--------------------+--------------------+---------------------+
+| Odoo 15.2     | Not certified      | Not certified      | Not certified       |
++---------------+--------------------+--------------------+---------------------+
+| Odoo 15.1     | Not certified      | Not certified      | Not certified       |
```

Comment:
We don't support 15.1 anymore, so i don't think it's useful to mention that 15.1 is not certified.